ros2param/test/test_verb_dump.py

```diff
@@ -115,7 +115,6 @@ def generate_test_description(rmw_implementation):
         ),
     ])
 
-
```

Comment:
AFAIS, this seems to be an intention to have 2 empty lines between classes or methods?

Reply:
No, it's my error. I'll fix this in the next commit